rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.7.2-release-notes.adoc

```diff
@@ -0,0 +1,62 @@
+= RPA Builder 6.7.2 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== September 27, 2024
+ 
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+//new action steps 
+
+//enhancements to action steps
+* The RPA Builder setup file now includes a digital signature to prevent security warnings. 
+//other updates 
+* The Google Chrome driver is updated to version 130. 
+* The MS Edge driver is updated to version 129.
```

Comment:
```suggestion

* The `StNetSh.exe` tool, part of the RPA Builder setup, now includes a digital signature to prevent security warnings. 
* The Google Chrome driver is updated to version 130. 
* The MS Edge driver is updated to version 129.
```